pkg/plugins/manager/manager_test.go

```diff
@@ -40,7 +40,7 @@ func TestPluginManager_Init(t *testing.T) {
 
 		assert.Empty(t, pm.scanningErrors)
 		verifyCorePluginCatalogue(t, pm)
-		verifyBundledPluginCatalogue(t, pm)
+		verifyBundledPlugins(t, pm)
```

Comment:
@wbrowne is it still worth doing this check? I renamed it to `verifyBundledPlugins` since we are not checking the plugins catalog.

Reply:
Yeah I think it's still valid as long as there is one there! 👍  on the rename